packages/mattermost-redux/src/actions/helpers.ts

```diff
@@ -63,19 +67,19 @@ export function requestFailure(type: ActionType, error: ServerError): any {
  * @returns {ActionFunc} ActionFunc
  */
 
-export function bindClientFunc({
+export function bindClientFunc<Data=any>({
```

Comment:
Should Data default to empty or would it make more sense for it to default to something like `never`? Alternatively, do we even need a default value for it? TS might be able to just infer this from `clientFunc` which would mean that we don't need a default

Reply:
check removing of default